fpga/lib/pcie_card.cpp

```diff
@@ -24,8 +24,8 @@ using namespace villas::fpga;
 // Instantiate factory to register
 static PCIeCardFactory PCIeCardFactoryInstance;
 
-static const kernel::pci::Device
-    defaultFilter((kernel::pci::Id(FPGA_PCI_VID_XILINX, FPGA_PCI_PID_VFPGA)));
+static const kernel::devices::PciDevice defaultFilter(
+    (kernel::devices::Id(FPGA_PCI_VID_XILINX, FPGA_PCI_PID_VFPGA)));
```

Comment:
Isnt there one `(` too much?

Reply:
Could you fix it?

Reply:
Nevermind, I am gonna merge this now. It would be nice if it could be addresses in a subsequent MR